Wikipedia/Code/Panels.swift

```diff
@@ -193,6 +193,11 @@ extension UIViewController {
         }
         let panel = SyncEnabledPanelViewController(showCloseButton: true, primaryButtonTapHandler: primaryButtonTapHandler, secondaryButtonTapHandler: nil, dismissHandler: nil, theme: theme)
         let presenter = self.presentedViewController ?? self
+        if presenter is WMFThemeableNavigationController {
```

Comment:
Would it be possible to move this `if` to the place where the problematic call to `wmf_showSyncEnabledPanelOncePerLogin` is happening so it doesn't even dip into this func if we don't want it to? (No worries if not.)